Documentation/ColumnsConfig/Type/Select/Properties/ForeignTableWhere.rst

```diff
@@ -13,7 +13,11 @@ foreign_table_where
    :RenderType: all
 
    The items from :ref:`foreign_table <columns-select-properties-foreign-table>`
-   are selected with this WHERE-clause.
+   are selected with this WHERE-clause. The WHERE-clause is effectively
```

Comment:
"WHERE clause" please without dash. Also I suggest to embed WHERE in the text role: :sql:

Reply:
ok, but this is how it is currently on the page.

Reply:
done.

Reply:
Yup, I know :-( - but we should make it better if we rework a section.